packages/react-server/src/ReactFlightServer.js

```diff
@@ -365,6 +417,17 @@ function describeObjectForErrorMessage(
   }
 }
 
+function isReactElement(value: mixed) {
```

Comment:
Let's revert this back to inline. Sometimes inlining is useful info for the VM because instead of checking the hidden class it can do it once in that one function. Like if it's going to run all these checks, afterwards it'll know what type it is. Otherwise the other function still has to check it.